pkg/resources/satellite/pod/node-pod.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - mountPath: /usr/src
           name: usr-src
           readOnly: true
+        - mountPath: /usr/lib
+          name: usr-lib
+          readOnly: true
```

Comment:
I wonder why this is needed for debian, but not for any of the other distributions. They should all have `/lib -> /usr/lib` links :thinking:

Reply:
:bulb: That's the missing puzzle piece. So we need additional tools because debian splits some things off into `/usr/lib/linux-kbuild-X`. On the other distros, `/lib/modules` and `/usr/src` is enough.